### DISCUSSION TITLE
Dark theme in cytoscape.js website for better experience

### DISCUSSION THREAD

**Discussion by DevyanshuNegi** [cytoscape/cytoscape.js#3311] [Ideas]
Adding a dark theme to the website would be very helpful for many users that work in night or even users working in the day but who prefer dark theme. because there is large part of the website that is in white color 
[js.cytoscape.org](https://js.cytoscape.org/)
Being a web developer I really like to add / work in dark theme website

![image](https://github.com/user-attachments/assets/26bf1b8a-1b4e-4ea4-a32b-657f0cdb8f9a)

**Comment by maxkfranz** [comment]
Yes, some people really like dark mode.  PRs welcome!
